src/main/java/com/sarahhand/fractals/juliaset/JuliaConfig.java

```diff
@@ -34,6 +46,7 @@
 	private int maxDwell = MAX_DWELL_DEFAULT;
 	private ColorPalette palette = ColorPalette.DEFAULT_PALETTE;
 	private ColorScheme colorScheme = COLOR_SCHEME_DEFAULT;
+	private Map<String,Float> constants = DEFAULT_CONSTANTS;
```

Comment:
Can you include a comment as to what constants are supported for JuliaSet - what "keys" are valid to put in the map?